cmd/stencil/stencil.go

```diff
@@ -55,6 +56,11 @@ func main() {
 		Action: func(c *cli.Context) error {
 			log.Infof("stencil %s", oapp.Version)
 
+			if c.Bool("debug") {
+				log.SetLevel(logrus.DebugLevel)
+				log.Debug("Debug logging enabled")
+			}
```

Comment:
This is probably something we should be doing by default in bootstrap, will add this later.